dvc/output/local.py

```diff
@@ -133,3 +140,24 @@ def save(self):
                 self.repo.scm.ignore(self.path)
 
         self.info = self.remote.save_info(self.path_info)
+
+    def get_dir_cache(self):
+        if not self.dir_cache or self.checksum_changed():
+            self.dir_cache = self.repo.cache.local.load_dir_cache(
+                self.checksum
+            )
+            self.cached_checksum = self.checksum
+
+        return self.dir_cache
+
+    def get_files_number(self):
+        if self.cache is None:
+            return 0
+
+        if self.is_dir_cache:
+            return len(self.get_dir_cache())
+        else:
+            return 1
+
+    def checksum_changed(self):
```

Comment:
This name is easy to confuse with a function that is checking that checksum of an output is changed compared to what is written in the dvc file.